extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -92,8 +92,9 @@ class AmpAccordion extends PreactBaseElement {
   /** @override */
   isLayoutSupported(unusedLayout) {
     userAssert(
-      isExperimentOn(this.win, 'amp-accordion-bento'),
-      'expected amp-accordion-bento experiment to be enabled'
+      isExperimentOn(this.win, 'bento') ||
+        isExperimentOn(this.win, 'bento-accordion'),
+      'expected "bento" or "bento-accordion" experiment to be enabled'
```

Comment:
Not sure if people will know which to use from this warning message. What do you think of this suggestion?
```suggestion
      'expected global "bento" or specific "bento-accordion" experiment to be enabled'
```

Alternatively
 - only include the specific one
 - also include an amp.dev link that would explain the difference between using one or the other

Reply:
Good idea. I modified the message as you suggest. And we can also provide a link when we have one.